### DISCUSSION TITLE
Using swup with Webflow & Framer

### DISCUSSION THREAD

**Discussion by ShaneSheppard** [swup/swup#839] [Feature ideas]
### Describe the problem 🧐

No clear instructions when using swup with no code builders like webflow or framer.
Which is a shame as i prefer this library to others.

### Describe the propsed solution 😎

Documentation on the Github general section in discussions. 

### Alternatives considered 🤔

This will gain more interest in your library as a lot of others are using barba.js as the alternative and this requires additional code form libraries like GSAP etc

### How important is this feature to you? 🧭

Would make my life a lot easier

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for related problems
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment]
We (the maintainers) currently don't have much expertise in using it with no-code builders, so the documentation in this area is limited. Feel free to share which information you feel is missing specifically, or add it to the [docs repository](https://github.com/swup/docs) yourself — we're open to contributions via pull requests. Starting this discussion is a nice first step for other users to chime in and share their knowledge, too.

Technically, page transitions are complex to pull off as they need be customized to the layout and structure of each individual site. I can't imagine a solution where integrating swup (or barba) can be done without touching code. If it happens, it'll probably be an integration created by webflow as part of a pre-made template.

Other than that, I'd suggest asking in the [Webflow Forum](https://discourse.webflow.com/), there's a few topics on swup already.

**Reply by ShaneSheppard** [reply]
Thanks @daun 
I think what needs to be made clear in the documentation is how to set-up the code when working with such no-code tools, as a lot of clients want said tools as backend solutions for commercial sites.
I'm fully aware that custom code will be required, but integrating the library with a tool such as Webflow is very under documented even when reviewing specific platform forums and Github discussions. 
Guidance on integration with different platforms and the specific nuances will gain the library grater adoption.

**Reply by hirasso** [reply]
@ShaneSheppard As you now have experience with integrating swup with Webflow, we would appreciate you contributing to the [docs](https://github.com/swup/docs). We are not a paid service, but an open source project. We depend on people like you to contribute. You feel like something's missing? Add it! :)

**Reply by ShaneSheppard** [reply]
@hirasso No problem i will put something together at least as a starting point for Webflow users.
